ModelBundle/Validator/Constraints/CheckMainAliasPresenceValidator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Validator\Constraints;
+
+use OpenOrchestra\ModelInterface\Model\SiteInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class CheckMainAliasPresenceValidator extends ConstraintValidator
+{
+    /**
+     * Checks if the passed value is valid.
+     *
+     * @param SiteInterface $value The value that should be validated
+     * @param Constraint    $constraint The constraint for the validation
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$value->getMainAlias()->isMain()) {
```

Comment:
``isMain()`` is useless : with ``getMainAlias`` you already get an alias flagged as main (https://github.com/open-orchestra/open-orchestra-model-bundle/blob/master/ModelBundle/Document/Site.php#L271)

But the code is the model has an error, the line 276 should be rewrote into ``return null`` (https://github.com/open-orchestra/open-orchestra-model-bundle/blob/master/ModelBundle/Document/Site.php#L276)